Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1196,14 +1196,14 @@ public function isBinaryOperator($index)
                 T_XOR_EQUAL => true,            // ^=
             );
             if (defined('T_POW')) {
-                $arrayOperators[T_POW] = true;          // **
-                $arrayOperators[T_POW_EQUAL] = true;    // **=
+                $arrayOperators[T_POW] = true; // **
+                $arrayOperators[T_POW_EQUAL] = true; // **=
```

Comment:
ok. didn't know that it is part of PSR-2.
But then the fixer should be in PSR-2 level, not symfony, right?